pkg/gpu/consumer.go

```diff
@@ -254,13 +263,32 @@ func (c *cudaEventConsumer) handleVisibleDevicesSet(vds *gpuebpf.CudaVisibleDevi
 	c.sysCtx.setUpdatedVisibleDevicesEnvVar(int(pid), unix.ByteSliceToString(vds.Devices[:]))
 }
 
+func (c *cudaEventConsumer) handleDeviceSync(event *gpuebpf.CudaSetDeviceEvent) error {
+	streams, err := c.streamHandlers.getActiveDeviceStreams(&event.Header)
+	if err != nil {
+		return err
```

Comment:
Can we wrap this error? 

```suggestion
		return fmt.Errorf("cannot get streams for the active device: %w", err)
```